src/ConfigInterface.php

```diff
@@ -97,7 +88,7 @@ public function getUsingCache();
     /**
      * Adds a suite of custom fixers.
      *
-     * @param iterable|Traversable|FixerInterface[] $fixers
+     * @param iterable|\Traversable|FixerInterface[] $fixers
```

Comment:
defining the namespace helps out SCA since these can now resolve the type for sure and no longer have to assume the default namespace